.mergify.yml

```diff
@@ -0,0 +1,101 @@
+pull_request_rules:
+  - name: automatic merge for Dependabot pull request that pass CI
+    conditions:
+      - author=dependabot[bot]
+      - check-success=build-workflow-complete
+    actions:
+      merge:
+        method: merge
+
+  # REVIEW MANAGEMENT
+
+  - name: ask alessandrod to review public API changes
+    conditions:
+      - files=xtask/public-api/aya.txt
+      - -closed
+    actions:
+      comment:
+        message: "Hey @alessandrod, this pull request changes the Aya Public API and requires your review."
```

Comment:
my point here was to drop "Aya", but I see now that this is only applied to aya, and not any of the other crates, so maybe it's fine.